jdk/src/jdk.jdwp.agent/share/native/libjdwp/debugInit.c

```diff
@@ -819,7 +810,7 @@ debugInit_reset(JNIEnv *env)
 
     currentSessionID++;
     initComplete = JNI_FALSE;
-    VMInitComplete = JNI_TRUE; /* The VM Is already initialized */
+    /* The VM Is already initialized so no need to set VMInitComplete = JNI_TRUE. */
```

Comment:
I don't think we want this comment in the code as it becomes another patch to maintain as the original code changes.